components/ucd/category/src/lib.rs

```diff
@@ -0,0 +1,313 @@
+// Copyright 2012-2015 The Rust Project Developers.
+// Copyright 2017 The UNIC Project Developers.
+//
+// See the COPYRIGHT file at the top-level directory of this distribution.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+
+#![deny(unsafe_code, missing_docs)]
+
+//! # UNIC — UCD — Category
+//!
+//! A component of [`unic`: Unicode and Internationalization Crates for Rust](/unic/).
+//!
+//! Unicode General Category.
+//!
+//!
+
+extern crate unic_ucd_core;
+
+use std::cmp::Ordering;
+
+use unic_ucd_core::UnicodeVersion;
+
+
+/// The [Unicode version](http://www.unicode.org/versions/) of data
+pub const UNICODE_VERSION: UnicodeVersion = include!("tables/unicode_version.rsv");
+
+/// Represents the Unicode Character
+/// [*General Category*](http://unicode.org/reports/tr44/#General_Category) property.
+///
+/// * <http://unicode.org/reports/tr44/#General_Category_Values>
+#[derive(Clone, Copy, Debug, PartialEq, Eq)]
+pub enum GeneralCategory {
+    /// An uppercase letter
+    ///
+    /// Abbreviated: Lu
```

Comment:
Too bad Rust's `enum`s don't get *alternate* names. We could really use them here in UCD packages.

Since it's markdown and will show up in docs, let use backticks for abbreviated names.

Also, you can `cargo doc` the package and check out the output and see if having a two-paragraph doc is okay here, or we better change it to one line, like:
```
An uppercase letter (Abbreviated name: `Lu`)
```